### ISSUE TITLE
Sentence doesn't make sense

### ISSUE THREAD

**Issue by p2635** [Super-Panama-World/Resource-Guide#5]
> There are three ways that Cantonese is romanized. You’ll likely have to get use to all three. Older resources use Yale, newer use Jyutping, natives often use the third. 

What do you mean natives often use the third?

**Comment by dddlr** [comment]
God I have no idea either

**Comment by Aryailia** [comment]
I'll rephrase it to something along the lines of 'they just do whatever'.

Cantonese natives in Hong Kong, Guandong, and any other regions  (which is meant to refer to the general populace), as with essentially every topolect/language that isn't standardised (so for Chinese languages probably every topolect with perhaps exception for Taiwan's Hakka and Taiwanese/Hokkien since it's government supported), is just people doing it by ear, which is highly affected by languages they know (like Pinyin) and languages learn.

The Hong Kong government does use a system whose English name is 'Cantonese Pinyin' (I believe used in some paper work, signs, etc.), however this is not used by regular people. When typing in Cantonese (which is distinct from typing in Mandarin), people typically use Sucheng (速成) which is just a simplified form of Cangjie or Cangjie itself (which is a shaped-based input method). Looking at the Wikipedia article for Cantonese Pinyin, it seems it has some traction in universities in mainland. There is some more local level governments in China that are also taking up the call to perserve local topolects, so perhaps there is some sense of standarisation on that front, but I'm not informed enough.

**Comment by p2635** [comment]
Ok... as long as the resources guide is fixed, I don't really care to be honest!

I guess you can add that to a more relevant part of the guide?

**Comment by Aryailia** [comment]
Added it to underneath guides/textbook section.
